lib/uvm/bridge-client.js

```diff
@@ -27,13 +37,13 @@ bridge = (function (emit, jsonStringify, arrayProtoSlice) {
             this._events[name].push(listener);
         }
     };
-}(__uvm_emit, JSON.stringify, Array.prototype.slice));
+}(__uvm_emit, CircularJSON.stringify, Array.prototype.slice));
 
 __uvm_dispatch = (function (bridge, bridgeEmit, jsonParse) {
     return function (args) {
         bridgeEmit.apply(bridge, jsonParse(args));
     };
-}(bridge, bridge.emit, JSON.parse));
-
+}(bridge, bridge.emit, CircularJSON.parse));
+delete CircularJSON;
```

Comment:
Right. Stupid mistake from my end. Removing the `var` will cause `delete` to work AFAIK.